### DISCUSSION TITLE
66 signups and... Discord?!

### DISCUSSION THREAD

**Discussion by ykdojo** [ykdojo/the-dojo-way#71] [Announcements]
66 people have signed up so far on README 🙌

Honestly, I never expected that many people to sign up... thank you so much!!

We don't have a dedicated discord group, and I don't think we will anytime soon... but!

We do have a discord for one of our other open source projects, [defaang](https://github.com/ykdojo/defaang). So if you'd like, feel free to find a join link [here](https://github.com/ykdojo/defaang) and come say hi 👋

**Comment by subhoghoshX** [comment]
I'm glad to see so many people signing up. If it's okay with you, you can add me as a maintainer. I can assist with merging some PRs in my spare time.

**Reply by ykdojo** [reply]
Thank you @subhoghoshX! I'll add you as a maintainer, then 😄
